Calculator/Calculator/CalculateItem.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CalculateItem.swift
+//  Calculator
+//
+//  Created by idinaloq on 2023/05/29.
+//
+
+protocol CalculateItem {
+    
+}
```

Comment:
```suggestion
protocol CalculateItem {}
```

빈 구현의 경우 중괄호를 붙이는 것이 일반적인 것 같습니다.